include/mysql/plugin_clone.h

```diff
@@ -69,6 +69,8 @@ struct Mysql_clone {
   int (*clone_server)(THD *thd, MYSQL_SOCKET socket);
 };
 
+extern const char *clone_compression_algorithms[3];
```

Comment:
See the other comment about `clone.h`.
Also 
```suggestion
extern const char *clone_compression_algorithms[];
```
(the 3 is dropped by the language)

Reply:
I am afraid we will hit compiler error if we don't specify the array size here. The clone_compression_algorithm array will be used in array_elements. It requires a fixed size for the  parameter.

```
template <class T, size_t N>
constexpr size_t array_elements(T (&)[N]) noexcept {
  return N;
}
```